astro/src/content/quickstarts/quickstart-springboot-web.mdx

```diff
@@ -56,188 +40,290 @@ The login flow will look like this after FusionAuth is introduced.
+* Your client Id is `e9fdb985-9173-4e01-9d73-ac2d60d1dc8e`.
+* Your client secret is `super-secret-secret-that-should-be-regenerated-for-production`.
 * Your example username is `richard@example.com` and your password is `password`.
 * Your admin username is `admin@example.com` and your password is `password`.
-* Your fusionAuthBaseUrl is 'http://localhost:9011/'
+* The base URL of FusionAuth is `http://localhost:9011/`.
 
-You can log into the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want, but with Docker/Kickstart you don't need to.
+You can log in to the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want to, but with Docker and Kickstart, everything will already be configured correctly.
 
-## Create your Spring Boot Application with Thymeleaf
+<Aside type="caution">
+  The `.env` and `kickstart.json` files contain passwords. In a real application, always add these files to your `.gitignore` file and never commit secrets to version control.
+</Aside>
 
-### Overview of the Application
+### Create A Basic Spring Boot Application
 
-We are going to be building a version of the ChangeBank app. This app will allow you to log in and see your "balance", as well as present a page that you can input a dollar amount and have that broken into change of dimes and pennies. You won't be keeping track of any account state beyond the user in FusionAuth, so the only actual functionality you will implement is the api call to make change.
+Next, you'll set up a basic Spring Boot project with a single app. While this guide builds a new Spring Boot project, you can use the same method to integrate your existing project with FusionAuth.
 
-You are going to build an application with three views:
-- `Home`: This is the welcome page than an unauthenticated user will see when first visiting your application.
-- `Account`: This is a protected view the user can only see after they have logged in.
-- `Make Change`: This view is also protected and demonstrates making an authenticated form post to the backend.
+#### Get The Template
 
-### Downloading the template from Spring Initializr
+Go to the Spring Initializr site at https://start.spring.io/ and download your own starter package. Choose the following options.
 
-We have already done this for you in the example repository, but if you wish to do so yourself you can go to the Initializr site at https://start.spring.io/ and download your own starter package. We will rely on three dependencies for this project:
+* Project - `Mavern`
+* Language - `Java`
+* Spring Boot Version - `2.7.16`
+* Project Metadata
+    * Group - `io.fusionauth.quickstart`
+    * Artifact - `springweb`
+    * Package Name - `io.fusionauth.quickstart.springweb`
+    * Packaging - `Jar`
+    * Java Version - `17`
+
+This project has the following three dependencies which you will need to add by clicking the "Add Dependencies" button when creating the package on the Initializr site.
```

Comment:
@mark-robustelli can you confirm if you added these dependencies 

![CleanShot 2023-10-06 at 00 27 52@2x](https://github.com/FusionAuth/fusionauth-site/assets/31291528/588e890d-6fb6-46e4-829a-39dae579d259)

The error log you posted shows that the package that you downloaded on the Initializr site does not have the three dependencies listed here